### PR TITLE
docs: Rework naming convention and status description requirements

### PR DESCRIPTION
**Pull request by mattmaniak**
# Description

## Type(s) 

- [ ] Game page addition
- [x] Game page modification
- [x] Other wiki edit

## Verification 
Have you

- [x] Followed the [contribution guidelines?](https://github.com/Whisky-App/whisky-book?tab=readme-ov-file#please-read-all-steps-before-contributing)
- [x] Ran `./scripts/generate.mjs`?
- [x] Ran `./scripts/lint.mjs`?
- [x] Checked spelling and grammar?
- [x] (If applicable) Provided documentation for a game not working? (i.e. detailing what happens when the game doesn't work?)

Most likely the biggest PR of all times here but it addresses some things that make docs navigation, maintenance and lack of some details issues:
- Abbrevations are no longer allowed and present. As the game titles are sorted alphabetically via markdown files names it's getting messier and messier with a proper ordering - some titles were completely somewhere else than expected. Also with a growing database of tested games, acronyms make things harder to maintain. Not everyone of us knows each acronym of each game and while we can easiliy search them via code editor it still gets messy when some markdown files are acronyms, some are partial acronyms and some are not at all. This PR provides a consistency of no acronyms with an exception of "goty" - Game of the Year which is well known.
- Every know issue occurence that was listed before on a game page was moved under a `[!WARNING]` heading to keep any games issues in one place.
- Names of the games were updated and taken directly from Steam for consistency. An idea has started [here](https://github.com/Whisky-App/whisky-book/pull/116#discussion_r1823226000). It may be automated in a future using a Steam API most likely where a [GitHub Actions would check for the game name using a given Steam app ID](https://github.com/orgs/Whisky-App/projects/1/views/1?pane=issue&itemId=85682132).
- Minor formatting issues were applied for a better consistency and legibility.
- Nothing new has been added to games performance info apart of the broken `REDlauncher` in The Witcher 3.

Any maintainer feedback is welcome as it is a huge change and it is better to discuss it before deploying but I hope that my idea and implementation have a solid background.

### CODE REVIEW

**Comment by JoshuaBrest** [comment]
I can add a lint feature later.

**Comment by mattmaniak** [comment]
> I can add a lint feature later.

You're welcome :) @hahayupgit any thoughts on PR?

File: src/game-support/metal-gear-solid-5-phantom pain.md
Comment (hahayupgit) [inline src/game-support/metal-gear-solid-5-phantom pain.md:1]: Change filename to metal-gear-solid-5-phantom-pain.md

File: README.md [hunk 1]
```diff
@@ -44,14 +44,16 @@ Documentation for Whisky.
...
+   - Format your file in all lowercase, using only alphabetical, numerical, and dash characters. Do not use abbreviations as they mess up titles ordering with an exception of `goty` which stands for "Game of the Year" and is used among many titles and known as a standard. Use a dash sign (`-`) everytime a space occurs in the original title. Replace roman numerals with arabic ones to distinguish words from numbers. Do not insert any [articles](https://www.grammarly.com/blog/parts-of-speech/articles/) in a markdown file name (`a`, `the`). Remember to match a filename with a full game title (eg. from Steam) using above exceptions and restrictions. If a game is not listed on Steam then add the store/page name at the very end of the file like `<some-game>-gog` i.e:
    ```
-   Metal Gear Solid V: The Phantom Pain -> mgs-5.md
-   Persona 3 Reload -> p3r.md
-   Star Wars Jedi: Fallen Order -> sw-fallen-order.md
-   Geometry Wars 3: Dimensions Evolved -> gw3-dimensions-evolved.md
+   Diablo IV (Battle.net) -> diablo-4-battle-net
+   Geometry Wars 3: Dimensions Evolved -> geomatry-wars-3-de.md
+   Metal Gear Solid V: The Phantom Pain -> metal-gear-solid-5.md
```
Comment (hahayupgit) [inline README.md:51]: change to `metal-gear-solid-5-phantom-pain.md`

File: src/game-support/tom-clancys-rainbow-six-siege.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+# Tom Clancy's Rainbow Six Siege
 <!-- script:Aliases [
     "Rainbow Six Siege Steam",
     "Rainbow Six Siege"
 ] -->
 
 {{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
 
+> [!WARNING]
 Tom Clancy's Rainbow Six Siege is currently **unplayable** in Whisky.
```
Comment (hahayupgit) [inline src/game-support/tom-clancys-rainbow-six-siege.md:10]: add indent for warning tag

**Review by hahayupgit — COMMENTED** [review]
Overall looks good! Minor things I'm going to go in and fix, and then approve. Great job!!!!

**Review by hahayupgit — APPROVED** [review]
Fantastic PR!

**Pull request merged**